app/package-lock.json

```diff
@@ -0,0 +1,5994 @@
+{
```

Comment:
It seems like you used `npm i <package>`. Delete your package-lock and re install your node modules from yarn